### PR TITLE
initial version of a Placeholder for when no countervalues loaded yet

### PR DESCRIPTION
**Pull request by gre**

### CODE REVIEW

File: src/components/Placeholder.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components'
+
+export const PlaceholderLine = styled.div`
+  background-color: ${p => (p.dark ? '#C2C2C2' : '#D6D6D6')};
```
Comment (meriadec) [inline src/components/Placeholder.js]: we may have theme vars for that

Comment (gre) [reply src/components/Placeholder.js]: but it's a placholder

Comment (gre) [reply src/components/Placeholder.js]: and we don't have these colors

Comment (gre) [reply src/components/Placeholder.js]: but i can try

**Pull request merged**